metadata/metadata.yml

```diff
@@ -1,5 +1,5 @@
 metadata_version: 1
-name: aind-smartspim-cell-segmentation
+name: aind-smartspim-cell-segmentation_test
```

Comment:
Could you please change this name to aind-smartspim-cell-segmentation? Thanks!

Reply:
Fixed